test/read-only-cluster/mzcompose.py

```diff
@@ -7,6 +7,8 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
 
+# XXX TODO(benesch) XXX: fix this test for the new 0dt feature flag flow.
```

Comment:
I'm working on this now. I don't expect it to change anything else about the PR, so should be safe to start reviewing!

Reply:
I've updated this test for the new 0dt flow and renamed it to the "0dt" test rather than the "read-only-cluster" test.